### PR TITLE
longevity: authenticator support

### PR DESCRIPTION
**Pull request by amoskong**
User and password will be passed through c-s cmdline, and authenticator can be configured from yaml config.

### CODE REVIEW

File: tests/gce-longevity-1.7-1TB-7days.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 test_duration: 10080
-stress_cmd: cassandra-stress write cl=QUORUM duration=10080m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=1000 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..1000000000 -log interval=5
-stress_cmd_1: cassandra-stress counter_write cl=QUORUM duration=10080m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=1000 -pop seq=1..10000000
+stress_cmd: cassandra-stress write cl=QUORUM duration=10080m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native user=cassandra password=cassandra -rate threads=1000 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..1000000000 -log interval=5
+stress_cmd_1: cassandra-stress counter_write cl=QUORUM duration=10080m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native user=cassandra password=cassandra -rate threads=1000 -pop seq=1..10000000
 n_db_nodes: 4
```
Comment (roydahan) [inline tests/gce-longevity-1.7-1TB-7days.yaml]: Please separate it to a different longevity yaml (i.e. gce-longevoty-1TB-with-auth.yaml).

Comment (roydahan) [inline tests/gce-longevity-1.7-1TB-7days.yaml]: Same for the short one.

Comment (larisau) [inline tests/gce-longevity-1.7-1TB-7days.yaml]: looks good. 
I agree that it's better to have separate yaml files for authentication testing

Comment (amoskong) [inline tests/gce-longevity-1.7-1TB-7days.yaml]: I'm using variants in yaml, then we can avoid some repeated yaml files.

**Comment by amoskong** [comment]
I already merged the PR to https://github.com/scylladb/scylla-cluster-tests/commits/longevity

**Comment by amoskong** [comment]
The longevity branch had been merged to master. So let's close the PR.

**Comment by roydahan** [comment]
@amoskong, I think this entire PR is irrelevant and need to be reverted.

For longevity, we should have another yaml.file that is exactly like longevity but will be longevity-auth-and-encrypt.yaml.

This yaml will have the same c-s commands as regular longevity but will use two more parameters for c-s: user/password.

**Comment by amoskong** [comment]
I suggested to use variants to avoid the repeat files and maintain, we can use filter (`/run/auth/default` or `/run/auth/enable`) in different jobs.

**Comment by roydahan** [comment]
@amoskong, I think this one isn't relevant anymore.
We decided to use it in one of our tests and not as an option that duplicates the list of tests, am I wrong?

**Comment by roydahan** [comment]
@amoskong ping

**Comment by amoskong** [comment]
We already had a single yaml for longevity with auth enabled. So we can close this issue now.

**Pull request closed without merging**